### ISSUE TITLE
Scaleability 

### ISSUE THREAD

**Issue by chelsealiggatt** [maxdee/alc_freeliner#14]
Is it possible to introduce the ability to scale geometries in FreeLiner? I believe this feature would greatly enhance performance!

**Comment by maxdee** [comment]
Yes it would be helpful. The problem is how to implement such feature. Its the type of thing that might have to wait until I re write the whole geometry system, that way resizing/moving/etc would be built in, instead of hacked on top of everything else which is not a good idea.

**Comment by maxdee** [comment]
Potential options for loading geometries and their potential commands : 
1. Load as is `geom load file.svg`
2. Load but align on center `geom load center file.svg`
3. Load, scale one axis to fit, and align on center `geom load center scale file.svg`
4. Load, scale by %, and align on center `geom load center scale 50 file.svg`
Would this be helpfull?
